tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -132,15 +136,15 @@
    "source": [
     "Whilst jupyter notebooks can be useful for demonstrating python code and prototyping new ideas, the main way in which to run python code is to execute a script. To do this, you need to write the python instructions within a file and execute it via the python interpretor.\n",
     "\n",
-    "Let's write our first python script:\n",
-    "1. In a new terminal, open a text editor (e.g. gedit; type `gedit hello.py` in the terminal and press enter; the text editor will open in a new window).\n",
-    "2. Replicating the previous exercise, type `print('Hello world')` into the editor and save the file.\n",
-    "3. Close gedit.\n",
-    "4. Type `python hello.py` to run the script.\n",
+    "For example if you were to write the following in a file (let's assume we call it hello.py):\n",
```

Comment:
@RMeli I ended up doing this which I think might be a suitable compromise.

I'm providing it as an example rather than something they should be doing it. This way if someone is really curious they can try it out themselves?

What'd you think?